source/slang/check.cpp

```diff
@@ -4742,21 +4742,119 @@ namespace Slang
             return result;
         }
 
+        /// Does there exist an implicit conversion from `fromType` to `toType`?
+        bool canConvertImplicitly(
+            RefPtr<Type> toType,
+            RefPtr<Type> fromType)
+        {
```

Comment:
Wouldn't it preferable to have pointers just passed as Type*? Preferable because we don't have the cost of ref counting - with no loss of generality. We never need the ref count - not only here because the function will never own the parameters, but even if it did - we can always take a ref on it, as our base class owns the ref count.

Reply:
Yes, in this case switching to `Type*` would be a win-win.